test/verify/check-machines

```diff
@@ -1566,10 +1566,13 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
 
         # We don't want to use start_vm == False because if we get a separate install phase
         # virt-install will overwrite our changes.
-        # https://bugzilla.redhat.com/show_bug.cgi?id=1818089
-        # After shutting it off virt-install will restart the domain and exit because of the above bug
-        # Wait for virt-install to define the VM and then stop it in real
-        self.machine.execute("virsh destroy pxe-guest")
+
+        if self.machine.image not in [ "fedora-33" ]:
```

Comment:
Eventually we want to invert the condition (as the fix trickles into more OSes), and perhaps factor it out. But we will notice when the fix trickles into other OSes, so we won't forget about this.